Fix Broken Package when Not Supplying Custom State Storage

matthemsteger opened the pull request:
I missed the closure and broke this when not using a customStorage, this fixes that. Sorry about that!

matthemsteger committed:
Missed stateStorage closure

I missed the closure and broke this when not using a customStorage, this fixes that

faceyspacey commented:
np.

can you use this command:

https://github.com/faceyspacey/redux-first-router-restore-scroll/blob/master/package.json#L14

to make the commit. it will format it in a way that auto-publishes once i merge it.

just add a new line somewhere or something and commit it.

matthemsteger committed:
fix($component): fix error when not using custom stateStorage

The custom state storage introduced a bug where when not using custom state storage, undfined was
used as stateStorage when setting prevKey

matthemsteger commented:
Hopefully an additional commit works?

faceyspacey commented:
yup, gracias.
